src/reports/AppointmentResourcePlanning/AppointmentResourcePlanning.test.tsx

```diff
@@ -47,85 +47,87 @@ const reportDataReturned: AppointmentResourcePlanningReportData[] = [
     }
 ];
 
+const questionnairesReturned = ["LMS2101_AA1", "LMS2101_BB1", "LMS2101_CC1"]
 
 const christmasEve97 = "1997-12-24";
 
-describe("appointment resource planning report with data", () => {
-    afterEach(() => {
-        MockDate.reset();
-    });
-
-    beforeEach(() => {
-        MockDate.set(new Date(christmasEve97));
-        mockAdapter.reset();
-        mockAdapter.onPost("/api/reports/appointment-resource-planning").reply(200, reportDataReturned);
-        mockAdapter.onPost("/api/reports/appointment-resource-planning-summary").reply(200, ReportSummary);
-    });
-
-    it("matches snapshot", async () => {
-        const history = createMemoryHistory();
-        const wrapper = render(
-            <Router history={history}>
-                <AppointmentResourcePlanning />
-            </Router>
-        );
-        await act(async () => {
-            await flushPromises();
-        });
-        await waitFor(() => {
-            expect(wrapper).toMatchSnapshot();
-        });
-    });
-
-    it("renders correctly", async () => {
-        const history = createMemoryHistory();
-        await act(async () => {
-            render(
-                <Router history={history}>
-                    <AppointmentResourcePlanning />
-                </Router>
-            );
-        });
-        expect(screen.queryByText("Run appointment resource planning report")).toBeVisible();
-        expect(screen.queryByText("Run a Daybatch first to obtain the most accurate results.")).toBeVisible();
-        expect(screen.queryByText("Appointments that have already been attempted will not be displayed.")).toBeVisible();
+// describe("appointment resource planning report with data", () => {
```

Comment:
Is it work keeping this at all? Maybe delete it as a separate commit and someone can revert if if they want it back?